examples/magic_image_arithmetic.py

```diff
@@ -0,0 +1,33 @@
+"""Basic example of using magicgui to create an Image Arithmetic GUI in napari."""
+import enum
+import numpy as np
+import napari
+
+
+# Enums are a convenient way to get a dropdown menu
+class Operation(enum.Enum):
+    """A set of valid arithmetic operations for image_arithmetic."""
+    add = np.add
+    subtract = np.subtract
+    multiply = np.multiply
+    divide = np.divide
```

Comment:
Silly question maybe, but what happens if a user tries diving by an image containing zeros? Does it get handled nicely or will that cause a problem?

Reply:
Well, 0/0 will give nan and then I think vispy becomes unhappy. See vispy/vispy#1509 and vispy/vispy#1511 for unfinished solutions to this problem.